code/game/objects/effects/spawners/random/trash.dm

```diff
@@ -0,0 +1,237 @@
+/obj/effect/spawner/random/trash
+	name = "trash spawner"
+	desc = "Ewwwwwww gross."
+	icon_state = "trash"
+
+/obj/effect/spawner/random/trash/garbage
+	name = "garbage spawner"
+	loot = list(
+		/obj/effect/spawner/random/trash/food_packaging = 56,
+		/obj/item/trash/can = 8,
+		/obj/item/shard = 8,
+		/obj/effect/spawner/random/trash/botanical_waste = 8,
+		/obj/effect/spawner/random/trash/cigbutt = 8,
+		/obj/item/reagent_containers/syringe = 5,
+		/obj/item/light/tube/broken = 3,
+		/obj/item/food/deadmouse = 2,
+		/obj/item/light/tube/broken = 1,
+		/obj/item/trash/candle = 1,
+	)
+
+/obj/effect/spawner/random/trash/cigbutt
+	name = "cigarette butt spawner"
+	loot = list(
+		/obj/item/cigbutt = 65,
+		/obj/item/cigbutt/roach = 20,
+		/obj/item/cigbutt/cigarbutt = 15,
+	)
+
+/obj/effect/spawner/random/trash/food_packaging
+	name = "empty food packaging spawner"
+	loot = list(
+		/obj/item/trash/raisins = 20,
+		/obj/item/trash/cheesie = 10,
+		/obj/item/trash/candy = 10,
+		/obj/item/trash/chips = 10,
+		/obj/item/trash/sosjerky = 10,
+		/obj/item/trash/pistachios = 10,
+		/obj/item/trash/boritos = 8,
+		/obj/item/trash/can/food/beans = 6,
+		/obj/item/trash/popcorn = 5,
+		/obj/item/trash/energybar = 5,
+		/obj/item/trash/can/food/peaches/maint = 4,
+		/obj/item/trash/semki = 2,
+	)
+
+/obj/effect/spawner/random/trash/botanical_waste
+	name = "botanical waste spawner"
+	icon_state = "peel"
+	loot = list(
+		/obj/item/grown/bananapeel = 6,
+		/obj/item/grown/corncob = 3,
+		/obj/item/food/grown/bungopit = 1,
+	)
+
+/obj/effect/spawner/random/trash/grille_or_waste
```

Comment:
ditto

Reply:
Updated grille_or_waste spawner to use other spawners instead of manually referencing the paths.  

https://github.com/tgstation/tgstation/pull/60522/commits/46f061b74afe97af05fa3aa2e4574bafefe82657

Thoughts?

Reply:
makes sense to me, it's easier to update